libraries/AP_Motors/AP_MotorsHeli_RSC.h

```diff
@@ -160,6 +163,9 @@ class AP_MotorsHeli_RSC {
     float           _fast_idle_timer;             // cooldown timer variable
     uint8_t         _governor_fault_count;        // variable for tracking governor speed sensor faults
     float           _governor_torque_reference;   // governor reference for load calculations
+    bool            _autorotating;
```

Comment:
we should add comments to the right of these parameters.

Reply:
fixed.  Thanks!